keps/sig-instrumentation/4785-crdmetrics-controller/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: CRDMetrics Controller
+kep-number: 4785
+authors:
+  - "@rexagod"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+status: implementable
+creation-date: 2024-08-27
+reviewers:
+  - "@dgrisonnet"
+  - "@logicalhan"
+  - "@mrueg"
+approvers:
+  - "@dgrisonnet"
+  - "@logicalhan"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.32"
```

Comment:
I thought this wasn't targeted at a release?

Reply:
This isn't as this is 100% an external addition, but I believe the CI fails without a proper specification I believe? I'll try dropping this.

Reply:
Set these to empty values.